toolkit/tools/imagecustomizer/container/Dockerfile.mic-container

```diff
@@ -2,6 +2,6 @@ FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 RUN tdnf update -y && \
    tdnf install -y qemu-img rpm coreutils util-linux systemd openssl \
       sed createrepo_c squashfs-tools cdrkit parted e2fsprogs dosfstools \
-      xfsprogs zstd veritysetup grub2 grub2-pc
+      xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify systemd-boot efibootmgr
```

Comment:
I don't think these new packages exist in Mariner 2.0.

Reply:
We should actually migrate to `FROM mcr.microsoft.com/cbl-mariner/base/core:3.0`

Reply:
https://dev.azure.com/mariner-org/ECF/_sprints/taskboard/BMP%20Team/ECF/2412?System.AssignedTo=%40me&workitem=10292